lua/lazyvim/plugins/extras/coding/codeium.lua

```diff
@@ -0,0 +1,22 @@
+return {
+
+  -- codeium cmp source
+  {
+    "nvim-cmp",
+    dependencies = {
+      {
+        "jcdickinson/codeium.nvim",
+        opts = {},
+        config = function(_, opts)
+          require("codeium").setup(opts)
+        end,
+      },
+    },
+    ---@param opts cmp.ConfigSchema
+    opts = function(_, opts)
+      table.insert(opts.sources, 1, { name = "codeium", group_index = 2 })
```

Comment:
If you put it in `group_index = 2`, doesn't that mean it will only give suggestions if there are no suggestions from `nvim_lsp` and rest which are by default `group_index = 1`. Wouldn't it be better for the `codeium` source to work alongside `nvim_lsp` source? How does it work for you with this configuration?

Reply:
> If you put it in group_index = 2, doesn't that mean it will only give suggestions if there are no suggestions from nvim_lsp and rest which are by default group_index = 1

From my understanding of this setting, you're right, but the Copilot extra plugin is also using `group_index = 2`, so maybe that's why.

Reply:
[jcdickinson/codeium.nvim](https://github.com/jcdickinson/codeium.nvim) has just been archived